client/src/components/overview/info/index.jsx

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import getRatingData from '../../sharedComponents/lib/averageRating.js';
+import StarRating from '../../sharedComponents/StarRating.jsx';
+import Price from './price.jsx';
+
+function ProductInfo({ metaData, product, style }) {
+  const ratingData = getRatingData(metaData);
+  const href = 'http://localhost:3000/';
+  const twitterText = `Checkout this ${product.name} from Atelier! Get it at ${href}${product.id}`;
+
+  return (
+    <div id="ProductInfo">
+      <StarRating count={ratingData.ratings} rating={ratingData.averageRating} />
+      <h2 className="infoCategory">{product.category}</h2>
+      <h1 className="infoName">{product.name}</h1>
+      <Price original={style.original_price} sale={style.sale_price} />
+      <p className="infoDescription">{product.description}</p>
+      <div id="shareButtons">
+        <i
+          // eslint-disable-next-line no-undef
+          onClick={() => window.open(`https://twitter.com/intent/tweet?text=${twitterText}`)}
```

Comment:
our linter seems to have a problem with window. It seems to think it is undefined.

Reply:
There is a similar situation with the temporary alert on line 28 where the linter errors that alert is undefined

Reply:
Good to know, thanks for pointing it out